frontend/simple/views/Join.vue

```diff
@@ -1,122 +1,118 @@
-                {{contract.memberApprovalThreshold | toPercent}}
-              </div>
-              <div class="panel-block">
-                <strong><i18n>Percentage of members are required to remove a member</i18n></strong>
-              </div>
-              <div class="panel-block">
-                {{contract.memberRemovalThreshold | toPercent}}
-              </div>
-              <div class="panel-block">
-                <strong><i18n>Contribution Transparency</i18n></strong>
-              </div>
-              <div class="panel-block">
-                {{contract.contributionPrivacy}}
-              </div>
-            </div>
-          </div>
-          <div class="column is-one-quarter">
-            <table class="table is-bordered is-striped is-narrow is-fullwidth">
-            <thead>
-            <tr>
-              <th><i18n>Group Members</i18n></th>
-            </tr>
-            </thead>
-            <tbody>
-            <tr v-for="member in contract.members" class="member">
-              <td>
-                <div class="media">
-                  <div class="media-left">
-                    <p class="image is-64x64">
-                      <img :src="member.attributes.picture">
-                    </p>
-                  </div>
-                  <div class="media-content">
-                    <strong>{{member.attributes.name}}</strong>
-                  </div>
-                </div>
-              </td>
-            </tr>
-            </tbody>
-          </table>
-          </div>
-        </div>
+  <main class="gi-join-grid">
+    <div class="gi-join-grid-header">
+      <h1 class="has-text-grey">
+        You’ve been invited to join a group!
+      </h1>
+      <h2 class="title is-1 is-marginless">{{contract.groupName}}</h2>
+      <h3 class="is-size-5">{{contract.sharedValues}}</h3>
+    </div>
+    <div class="gi-join-grid-ctas">
+      <div class="buttons">
+        <button
+          class="button is-large is-primary"
+          data-test="acceptLink"
+          v-on:click="accept">
+          Join Group
+        </button>
+        <button
+          class="button is-large is-outlined"
+          data-test="declineLink"
+          v-on:click="decline">
+          No, thanks
+        </button>
       </div>
-    </section>
+      <p v-if="errorMsg" class="has-text-danger has-text-centered-mobile">{{errorMsg}}</p>
+    </div>
+    <div class="gi-join-grid-graphic" v-if="contract.members.length">
+      <members-circle :members="contract.members">
+        <bars
+          :currency="contract.incomeCurrencySign"
+          :history="contract.history"
+          :mincome="contract.incomeProvided" />
+      </members-circle>
+    </div>
+  </div>
+  </main>
 </template>
+<style lang="scss" scoped>
+@import "../sass/theme/index";
+@import "../sass/bulma_overrides/components/navbar";
+
+.gi-join {
+  &-grid {
+    display: grid;
+    grid-template-areas: "header" "graphic" "ctas";
+    grid-gap: $gi-spacer-lg;
```

Comment:
@taoeffect and @hubudibu  - Since GI supports [only modern/recent browsers](https://caniuse.com/#feat=css-grid), I've added [CSS Grid](https://css-tricks.com/snippets/css/complete-guide-grid/), so now Join and MembersCircle View are full responsive without using any Javascript. 

Note on mobile how MembersCircle show up before Join/Decline buttons as @taoeffect asked. Bulma grid is based on flexbox only, that is not powerful enough to handle this type of layout manipulation.

CSS Grid is awesome 🎉